### PR TITLE
Fix/simplify ux flow addition

### PR DESCRIPTION
**Pull request by krhoda**
Depends on #30.

Simplifies work to add new flow to UI changing nested ternary into switch statement.

Exposes all configurable variables used by Witness/Issuer Worker as secrets, then documents them in secrets.md. 

Needs some post-merge linking as well, will be taken care of after #28

### CODE REVIEW

File: demo/witness/README.md [hunk 1]
```diff
@@ -6,22 +6,11 @@ A Cloudflare Worker to act as a witness for Rebase claims.
...
-```bash
-wrangler secret put REBASE_SK
-wrangler secret put TWITTER_BEARER_TOKEN
-```
-
-> The private key is expected to be a JWK. You can generate one with
-> `didkit generate-ed25519-key`.
-
-> This key is also expected to have a corresponding `did:web` outlined
+The worker needs several secrets set using `wrangler secret put`, the details of which are outlined in this repository's `secrets.md` [found here]().
```
Comment (sbihel) [inline demo/witness/README.md:11]: missing link I believe

File: demo/witness/README.md [hunk 2]
```diff
@@ -6,22 +6,11 @@ A Cloudflare Worker to act as a witness for Rebase claims.
 
 Requires [Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update#additional-installation-instructions).
 
-You need a Cloudflare account, and the Account ID needs to go in
-the `wrangler.toml`.
+You need a Cloudflare account, and the Account ID needs to go in the `wrangler.toml`.
```
Comment (sbihel) [inline demo/witness/README.md:9]: should probably provide a wrangler.toml example that people should then cp and fill in the necessary fields

**Review by sbihel — APPROVED** [review]

File: demo/witness/README.md (hunk 2 above)
Comment (krhoda) [reply demo/witness/README.md:9]: There is one! `wrangler.toml.example`

File: demo/witness/README.md (hunk 1 above)
Comment (krhoda) [reply demo/witness/README.md:11]: It's a catch-22, `secrets.md` isn't in the repo yet.

**Comment by krhoda** [comment]
Inadvertently merged these changes into the SoundCloud PR. Closing this PR with the assumption the SoundCloud one will go through.

**Pull request closed without merging**